package.json

```diff
@@ -16,11 +16,12 @@
     "gulpplugin"
   ],
   "dependencies": {
-    "gulp-util": "^3.0.0",
     "lodash": "^4.12.0",
     "minimatch": "^3.0.3",
+    "plugin-error": "^0.1.2",
     "rcloader": "^0.2.2",
-    "through2": "^2.0.0"
+    "through2": "^2.0.0",
+    "vinyl": "^2.1.0"
```

Comment:
This should be a devDep, right?

Reply:
Thanks for catching it. You are right: `vinyl` is only used for the tests and should be a `devDependency`.

Reply:
We want to be careful with this because we don't want people inadvertently depending on vinyl if they aren't using it outside of DevDeps.

Reply:
Thanks @phated!